coala_quickstart/Extensions.py

```diff
@@ -0,0 +1,172 @@
+exts = {'.w': 'C',
+        '.cs': 'C#',
+        '.csx': 'C#',
+        '.c': 'C++',
+        '.c++': 'C++',
+        '.cpp': 'C++',
+        '.cxx': 'C++',
+        '.h': 'C++',
+        '.h++': 'C++',
+        '.hh': 'C++',
+        '.hpp': 'C++',
+        '.hxx': 'C++',
+        '.tcc': 'C++',
+        '.tpp': 'C++',
+        '.cmake': 'CMake',
+        '.cmake.in': 'CMake',
```

Comment:
This will need its own thing as I'm only checking extensions here (`CMakeLists.txt` will match `.txt`)